mopidy_local/ext.conf

```diff
@@ -1,6 +1,6 @@
 [local]
 enabled = true
-library = json
+library = sqlite
```

Comment:
Since the `local/library` config value is deprecated, this should have no effect. The line can be removed.